### DISCUSSION TITLE
How to increase the number of tokens/symbols in Response

### DISCUSSION THREAD

**Discussion by infinity-nft** [srcnalt/OpenAI-Unity#20] [Q&A] [answered]
Hello I have a question. Regardless of the MaxTokens parameter, the response from OpenAI seems short. How can I increase the number of tokens/characters in Response?

**Comment by srcnalt** [comment, marked as answer]
Hi @infinity-nft, responses of the API depends on the prompt as well as MaxTokens param. I sometimes use `Reply in maximum x words` as a part of the prompt to shorten it. You might reverse that and add it to your prompt as `Reply in minimum x words.` Let me know if that works.
